src/styles/event-organizer/frontend.pcss

```diff
@@ -6,20 +6,58 @@
 .tribe-block__organizer__details {
 	margin: 15px 0;
 	position: relative;
+	border-top: 1px solid #d5d5d5;
+	padding: 32px 0;
+
+	@media screen and (max-width: 480px) {
+		padding: 22.5px 0;
+	}
```

Comment:
Mobile first. Also, 480px is a small breakpoint, maybe aim for something like 768px?